pkg/operator/cassandra/controller/controller_test.go

```diff
@@ -41,7 +41,10 @@ func newFakeClusterController(kubeObjects []runtime.Object, rookObjects []runtim
 
 	// Add sample-controller types to the default Kubernetes Scheme so Events can be
 	// logged for sample-controller types.
-	rookScheme.AddToScheme(scheme.Scheme)
+	err := rookScheme.AddToScheme(scheme.Scheme)
+	if err != nil {
+		logger.Errorf("failed to add scheme. %v", err)
```

Comment:
We should `assert.NoError(t, err)` otherwise the rest of the test will be compromised.

Reply:
Done.